sdk/python/src/dagger/_codegen/generator.py

```diff
@@ -505,6 +505,8 @@ def doc(s: str) -> str:
     """Wrap string in docstring quotes."""
     if "\n" in s:
         s = f"{s}\n"
+    elif s.endswith('"'):
```

Comment:
Would the same problem happen if the doc strings began w/ `"` (for whatever reason)? If so may as well fix here too I guess.